### DISCUSSION TITLE
API support skip event?

### DISCUSSION THREAD

**Discussion by somedevreally** [opensubsonic/open-subsonic-api#81] [General]
Hello Devs,

is there a possibility to have a skip event or song scrobbled support? 
refering to this [discussion ](https://github.com/navidrome/navidrome/issues/50)
currently am using Navidrome server with Symfonium client on Android 

thanks

**Comment by lachlan-00** [comment]
Ampache supports play/skip but i've not seen it in subsonic servers

**Comment by dweymouth** [comment]
This would need to be an OpenSubsonic API extension with a new endpoint like `setSkipped` or something like that.

**Reply by somedevreally** [reply]
any plans to add this? lot of playlists will became more smarter :)
